applications/web/src/components/ToolBar.svelte

```diff
@@ -7,7 +7,6 @@
 
   let solving = false
   // todo ask Matt why is this a no-op?
```

Comment:
Some more things you could remove here

Reply:
And further down in the file too.

Reply:
@av8ta  plz review I have made few more changes.